qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -396,6 +477,20 @@ def _ramp_simultaneously(self) -> None:
         for slave in self.submodules.values():
             slave.ramp_to_target()
 
+    def _ramp_simultaneously_blocking(self) -> None:
+        """
+        Ramp all three fields to their target simultaneously at their given
+        ramp rates. NOTE: there is NO guarantee that this does not take you
+        out of your safe region. Use with care. This function is BLOCKING.
+        """
+        for slave in self.submodules.values():
+            slave.ramp_to_target()
```

Comment:
these two lines can be safely substituted with `self._ramp_simultaneously()`, right? This reduces code duplication, makes the code more readable.